inputs.py

```diff
@@ -52,14 +52,19 @@ def get_graph_files(input_dir,output_dir, kg_type):
         for k in list(existence_dict.keys()):
             for fname in os.listdir(input_dir):
                 if k in fname:
-                    if k == 'PheKnowLator_v3.0.2_full_instance_relationsOnly_OWLNETS_Triples_Identifiers':
-                        triples_list_file = input_dir + '/' + fname
-                    if k == 'PheKnowLator_v3.0.2_full_instance_relationsOnly_OWLNETS_NodeLabels':
-                        labels_file = input_dir + '/' + fname
                     if k == '_example_input':
```

Comment:
@lagillenwater  this is where we check for the _example_inputs in the input directory

Reply:
so the knowledge graph directory is in the input_dir?

Reply:
yes, and the name should match that of kg_type (specified by --knowledge-graph in the arguments), I showed an example of the input directory in the ReadMe if you take a look in edit mode, you can see the visual in the ## Output Structure at the bottom

Reply:
Ah, I see now. So those directories would stay the same and you would change the example input file. Thanks for the clarification.